src/main/kotlin/app/CacheService.kt

```diff
@@ -0,0 +1,84 @@
+@file:Suppress("SqlResolve")
+
+package app
+
+import app.util.HikariCpDataSource
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.slf4j.LoggerFactory
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+
+object CacheService {
+    private val log = LoggerFactory.getLogger(CacheService.javaClass)
+    private val objectMapper = jacksonObjectMapper()
+
+    private fun createTableIfAbsent() {
+        val connection = HikariCpDataSource.connection
+        val statement = connection.createStatement()
+
+        statement.execute(
+            "CREATE TABLE IF NOT EXISTS userinfo (" +
+            "id VARCHAR2 PRIMARY KEY, " +
+            "timestamp TIMESTAMP, " +
+            "data JSON" +
+            ")"
+        )
+    }
+
+    fun selectJsonFromDb(username: String): String? {
+        val connection = HikariCpDataSource.connection
+
+        createTableIfAbsent()
+
+        val preparedStatement = connection.prepareStatement(
```

Comment:
This could also be a multi-line string.

Reply:
https://github.com/Attacktive/profile-summary-for-github/commit/aea86489ef743a02e52ffb2c09dfacdd37f3157e